sunkit_image/conftest.py

```diff
@@ -7,7 +7,7 @@
 
 import sunpy.tests.helpers
 from sunpy.tests.hash import HASH_LIBRARY_NAME
-from sunpy.tests.helpers import new_hash_library, generate_figure_webpage
+# from sunpy.tests.helpers import new_hash_library, generate_figure_webpage
```

Comment:
I had to make this change because without this pytest was not working. If something else is wrong please point it out.

Reply:
We aren't using it for now. Just delete it.